resources/sdk/clisdkclient/extensions/models/implicitWebpage.go

```diff
@@ -0,0 +1,48 @@
+package models
+
+import (
+	"fmt"
+)
+
+type ImplicitWebpage struct {}
+
+func (w ImplicitWebpage) GetImplicitWebpage(clientID string, environment string, redirectURI string) string {
+	return fmt.Sprintf(AuthWebpage, clientID, redirectURI, environment)
+}
+
+const AuthWebpage = `
+<html>
+<head>
+    <title>Implicit Login</title>
+
+    <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
```

Comment:
@timsmithgenesys Couple of thoughts:

1.  Do we want to reference a third party CDN?  If the CDN is unavailable the user cant authenticate with the implicit grant?
2. GoLang now supports embedded resources.  Rather then putting a web page right here why not embedd the HTML and a local copy of the above javascript file right in the binary and distribute it with the binary?

Thoughts,
    John